Applications/DataExplorer/DataView/MshView.cpp

```diff
@@ -180,13 +186,37 @@ void MshView::openValuesEditDialog()
 	valueEdit.exec();
 }
 
+void MshView::openAddLayerDialog()
+{
+	QModelIndex const index = this->selectionModel()->currentIndex();
```

Comment:
"this" is not strictly needed but I use it to indicate class variables and methods of the Qt-GUI classes where our normal naming conventions don't apply or it's difficult to look up the definitions.